modules/types/src/basic.ts

```diff
@@ -0,0 +1,21 @@
+import { BigNumber as ethersBig } from "ethers/utils";
+
+export type Address = string;
+
+export type BigNumber = ethersBig;
+export const BigNumber = ethersBig;
+
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+export type SolidityValueType = any; // FIXME: use cf type
+
+export const ConnextEvents = {
+  RECIEVE_TRANSFER_FAILED: "receiveTransferFailedEvent",
+  RECIEVE_TRANSFER_FINISHED: "receiveTransferFinishedEvent",
+  RECIEVE_TRANSFER_STARTED: "receiveTransferStartedEvent",
+};
+export type ConnextEvent = keyof typeof ConnextEvents;
```

Comment:
we really need to extend the cf events as well (since the client emits those additionally)

Reply:
obviously not blocking the merge though, ill make an issue

Reply:
https://github.com/ConnextProject/indra/issues/525